foxpuppet/windows/browser/notifications/addons.py

```diff
@@ -34,8 +33,14 @@ def addon_name(self):
 
         """
         with self.selenium.context(self.selenium.CONTEXT_CHROME):
-            if self.window.firefox_version >= 55:
-                name = self.root.find_element(*self._addon_name_locator).text
+            if self.window.firefox_version >= 59:  # Bug 1431242
+                el = self.root.find_anonymous_element_by_attribute(
+                    'class', 'popup-notification-description')
+                return el.find_element(By.CSS_SELECTOR, 'b').text
```

Comment:
Oh this is how its done?? I couldn't figure out a way to find the element I totally forgot that it could be anonymous.